core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -14,7 +14,6 @@ import com.twitter.finagle.http.{
 }
 import com.twitter.finagle.http.exp.{Multipart => FinagleMultipart}
 import com.twitter.io.Buf
-import com.twitter.util.{Return, Throw, Try}
```

Comment:
Love it!

Reply:
Thank you for the love @vkostyukov :) . I'm glad to learn from you and contribute to this awesome piece of work. hope to contribute more :)